common.yaml.jinja

```diff
@@ -50,7 +50,7 @@ services:
   {%- if smtp_relay_host %}
 
   smtpreal:
-    image: tvial/docker-mailserver:latest
+    image: ghcr.io/docker-mailserver/docker-mailserver:stable
```

Comment:
Add a question to the questionary, to ask for the version you want to use here.

By default, it should be `8.0`, to avoid surprises, don't you think?

You can link to all the versions in the question if you find it useful: https://github.com/orgs/docker-mailserver/packages/container/docker-mailserver/versions

Reply:
Yes, thanks! I've added a question with that: https://github.com/Tecnativa/doodba-copier-template/compare/17b0a7860e3786ac0343c036cc96e5c812144079..6a6eb0f4cae1cb07aa03f8529550955bdf1a63c0#diff-a74d725f30a732f10530aec0b66cdc1f1856272e35ede5ba4826ead3075c3d3dR414-R424